write.go

```diff
@@ -315,50 +318,50 @@ func encodeElementHeader(w dicomio.Writer, t tag.Tag, vr string, vl uint32) erro
 	return nil
 }
 
-func writeValue(w dicomio.Writer, t tag.Tag, value Value, valueType ValueType, vr string, vl uint32,  opts ...WriteOption) error {
-		if vl == tag.VLUndefinedLength && valueType <= 2 { // strings, bytes or ints
-			return fmt.Errorf("Encoding undefined-length element not yet supported: %v", t)
-		}
-
-		// TODO floats?
-		v := value.GetValue()
-		switch valueType {
-		case Strings:
-			return writeStrings(w, v.([]string), vr)
-		case Bytes:
-			return writeBytes(w, v.([]byte), vr)
-		case Ints:
-			return writeInts(w, v.([]int), vr)
-		case PixelData:
-			return writePixelData(w, t, value, vr, vl)
-		case SequenceItem:
-			return writeSequenceItem(w, t, v.([]*Element), vr, vl, opts...)
-		case Sequences:
-			return writeSequence(w, t, v.([]*SequenceItemValue), vr, vl, opts...)
-		default:
-			return fmt.Errorf("ValueType not supported")
-		}
-		return fmt.Errorf("Something went real bad, this should never be reached")
+func writeValue(w dicomio.Writer, t tag.Tag, value Value, valueType ValueType, vr string, vl uint32, opts ...WriteOption) error {
+	if vl == tag.VLUndefinedLength && valueType <= 2 { // strings, bytes or ints
+		return fmt.Errorf("Encoding undefined-length element not yet supported: %v", t)
 	}
 
+	// TODO floats?
+	v := value.GetValue()
+	switch valueType {
+	case Strings:
+		return writeStrings(w, v.([]string), vr)
+	case Bytes:
+		return writeBytes(w, v.([]byte), vr)
+	case Ints:
+		return writeInts(w, v.([]int), vr)
+	case PixelData:
+		return writePixelData(w, t, value, vr, vl)
+	case SequenceItem:
+		return writeSequenceItem(w, t, v.([]*Element), vr, vl, opts...)
+	case Sequences:
+		return writeSequence(w, t, v.([]*SequenceItemValue), vr, vl, opts...)
+	default:
+		return fmt.Errorf("ValueType not supported")
+	}
+	return fmt.Errorf("Something went real bad, this should never be reached")
+}
+
 func writeStrings(w dicomio.Writer, values []string, vr string) error {
-		s := ""
-		for i, substr := range values {
-			if i > 0 {
-				s += "\\"
-			}
-			s += substr
+	s := ""
```

Comment:
we could consider using a strings.Builder for this: https://golang.org/pkg/strings/#Builder, but won't block the PR on it, more just to track.